templates/signed-out.html

```diff
@@ -1,11 +1,33 @@
-{% extends 'layouts/_base.html' %}
+{% extends 'errors/_base.html' %}
 
-{% set page_title = _("Signed out") %}
+{% set page_title = _("Save and sign out confirmation") %}
 
 {% block main %}
-  <h1>{{ _("Your survey answers have been saved. You are now signed out") }}.</h1>
 
-  {% if account_service_log_out_url %}
-    <a href="{{account_service_log_out_url}}">{{ _("Return to your account") }}</a>
+  {% from "components/panel/_macro.njk" import onsPanel %}
+  {{
+    onsPanel({
+      "id": '',
+      "type": 'success',
+      "iconType": 'check',
+      "iconSize": 'xl',
+      "classes": "ons-u-mb-xl",
+      "body": _("<h1>Your survey answers have been saved</h1>")
+    })
+  }}
+
+  {% if survey_type and (survey_type in SURVEY_TYPES_BUSINESS or survey_type in SURVEY_TYPES_DEFAULT) %}
+
+    {% if business_logout_url %}
```

Comment:
Currently when testing I do not see a `sign out link` on the header after save and exiting when testing on a business schema.